### PR TITLE
Move org-emphasis-regexp-components to spacemacs/view-org-file

### PR DESCRIPTION
**Pull request by tuhdo**
If we put this in Org layer, new users won't have that layer and thus
this fix will not be applied.

### CODE REVIEW

File: core/core-funcs.el [hunk 1]
```diff
@@ -128,6 +128,11 @@ Supported properties:
     (show-all))
    (t nil))
 
+  ;; Make ~SPC ,~ work, reference:
+  ;; http://stackoverflow.com/questions/24169333/how-can-i-emphasize-or-verbatim-quote-a-comma-in-org-mode
+  (setcar (nthcdr 2 org-emphasis-regexp-components) " \t\n")
+  (org-set-emph-re 'org-emphasis-regexp-components org-emphasis-regexp-components)
```
Comment (CestDiego) [inline core/core-funcs.el]: @tuhdo are you sure this commit is ok? there is no `SPC j` here

Comment (tuhdo) [inline core/core-funcs.el]: Opps I messed up the commit. This change should be another commit.

Comment (tuhdo) [inline core/core-funcs.el]: The commit the title was referring to should be here: https://github.com/syl20bnr/spacemacs/pull/1968

**Comment by syl20bnr** [comment]
Thank you again :-)
Cherry-picked into `develop` branch, you can safely delete your branch.

**Comment by CestDiego** [comment]
Wait, So are you telling me that we should think of users who don't have the org-layer present and users who will? ehmm. So all documentation stuff..should be in the core of spacemacs? as in related to org stuff... like babel for elisp, some variables. etc etc

**Comment by tuhdo** [comment]
@CestDiego Good point. I think `org-layer` should be in the core of Spacemacs, since it is configured to look so nicely.
